edx_name_affirmation/api.py

```diff
@@ -145,3 +146,69 @@ def update_verification_attempt_id(user, verification_attempt_id):
         )
     )
     log.info(log_msg)
+
+
+def update_is_verified_status(
```

Comment:
Just an observation, no need to change at the moment, but the function `update_verification_attempt_id` looks similar to this `update_is_verified_status` function to me. 
Maybe in the future, when we are done with frontend implementation, these two can be merged?